### ISSUE TITLE
PIVX site forked

### ISSUE THREAD

**Issue by Fuzzbawls** [PIVXMasterNodeDev/PIVXMasterNodesPro#12]
looks like you didn't update to v2.3.0 so now the stats are being pulled from a forked chain.

**Comment by Fuzzbawls** [comment]
Also, think you changed your slack name one or more times...can't find you there anymore.

**Comment by JSponaugle** [comment]
Yes i was just notified about the new wallet. I am in the processes of updating my CoreCode for Stats Sites. And Also Moving All Coin Data to ElasticSearch (Long Process) I have PIVX on the List and will Install a new wallet and Get that Fixed ASAP.

**Comment by Fuzzbawls** [comment]
I did send a preliminary notification via slack back before the v2.3.0 release, indicating that there was an upcoming mandatory update, and that changes to masternode information availability would need to be taken into consideration.

Upon official release of v2.3.0, I sent another notice via Slack. Both messages were sent in the PIVX Slack as that is where prior communication regarding the site had been conducted. If there is a more suitable means of communicating updates to you specifically, please let us know.
